README.md

```diff
@@ -87,19 +24,17 @@ This can be enabled by setting `enforce_trusted_endpoints` in your Grafana confi
 enforce_trusted_endpoints = true
 ```
 
-## Configuring the datasource in Grafana
+## Configure the Azure Data Explorer data source
 
-The steps for configuring the integration between the Azure Data Explorer service and Grafana are:
+To configure ADX for using this data source, create an Azure Active Directory (AAD) Application and AAD Service Principle, log into the Azure Data Explorer WebExplorer and connect the AAD Application to an Azure Data Explorer database user, and use the AAD Application to configure the data source connection in Grafana.
```

Comment:
I would actually suggest formatting each action as its own step, and I really think users would benefit instruction on how to actually accomplish these actions instead of just being told to do them unless it's super obvious. 

```suggestion
To configure ADX for using this data source:
1. Create an Azure Active Directory (AAD) Application and AAD Service Principle.
1. In the Azure Data Explorer WebExplorer and connect the AAD Application to an Azure Data Explorer database user.
1. Use the AAD Application to configure the data source connection in Grafana.
```